SayTheirNames/Source/Controller/More/MoreView.swift

```diff
@@ -135,17 +164,18 @@ final class MoreView: UIView {
         
         scrollView.anchor(superView: self, top: topAnchor, leading: leadingAnchor,
                           bottom: bottomAnchor, trailing: trailingAnchor, padding: .zero, size: .zero)
-        stack.anchor(superView: scrollView, top: scrollView.topAnchor, leading: nil,
-                           bottom: nil, trailing: nil,
+        stack.anchor(superView: scrollView, top: scrollView.topAnchor, leading: scrollView.leadingAnchor,
+                     bottom: scrollView.bottomAnchor, trailing: scrollView.trailingAnchor,
                            padding: UIEdgeInsets(top: 20, left: 20, bottom: 20, right: 20), size: .zero)
         
         NSLayoutConstraint.activate([
             moreCard.topAnchor.constraint(equalTo: scrollView.topAnchor, constant: 20),
             moreCard.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -20),
             moreCard.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 20),
-            moreCard.heightAnchor.constraint(equalToConstant: 150),
-            thankYouLabel.bottomAnchor.constraint(equalTo: scrollView.bottomAnchor, constant: -Theme.Components.Padding.large)
+            moreCard.heightAnchor.constraint(equalTo: moreCard.widthAnchor, multiplier: 0.6)
```

Comment:
mind reusing the padding from `Theme`?

Reply:
Should I define this multiplier in the theme?